pkgs/development/libraries/libpfm/default.nix

```diff
@@ -1,24 +1,34 @@
-{ lib, stdenv, fetchurl
+{ lib
+, stdenv
+, fetchurl
 , enableShared ? !stdenv.hostPlatform.isStatic
+, libgnurx
 }:
 
-stdenv.mkDerivation (rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "4.13.0";
   pname = "libpfm";
 
   src = fetchurl {
-    url = "mirror://sourceforge/perfmon2/libpfm4/${pname}-${version}.tar.gz";
+    url = "mirror://sourceforge/perfmon2/libpfm4/libpfm-${finalAttrs.version}.tar.gz";
     sha256 = "sha256-0YuXdkx1VSjBBR03bjNUXQ62DG6/hWgENoE/pbBMw9E=";
   };
 
+  # Don't install libpfm.so on windows as it doesn't exist
+  # This target is created only if `ifeq ($(SYS),Linux)` passes
+  patches = [ ./fix-windows.patch ];
```

Comment:
Upstream?

Reply:
Yeah, maybe worth opening an issue. It's a bit subtle though. [It tries to install SLIBPFM](https://github.com/wcohen/libpfm4/blob/70b5b4c82912471b43c7ddf0d1e450c4e0ef477e/lib/Makefile#L432), but `SLIBPFM` is [set only if we're on linux](https://github.com/wcohen/libpfm4/blob/70b5b4c82912471b43c7ddf0d1e450c4e0ef477e/lib/Makefile#L223-L228)